app/models/miq_action.rb

```diff
@@ -882,6 +886,7 @@ def action_custom_automation(action, rec, inputs)
     args = {
       :object_type      => rec.class.base_class.name,
       :object_id        => rec.id,
+      :user_id          => ae_user_object(rec).id,
```

Comment:
@lfu 
Do we need the group_id and tenant_id also to be included in here.
If you look in app/models/miq_request_task.rb deliver_to_automate you would see us
passing the user, group, tenant. This also happens in lib/miq_automation_engine/miq_ae_event.rb#call_automate